controllers/phish.go

```diff
@@ -195,6 +196,7 @@ func (ps *PhishingServer) ReportHandler(w http.ResponseWriter, r *http.Request)
 // (such as clicked link, etc.)
 func (ps *PhishingServer) PhishHandler(w http.ResponseWriter, r *http.Request) {
 	r, err := setupContext(r)
+	w.Header().Set("X-Server", config.ServerName) // Useful for checking if this is a GoPhish server (e.g. for campaign reporting plugins)
```

Comment:
Hey @glennzw,

Thanks for the quick follow-up! The last thing to knock out would be to just move this line below the `if err != nil` error check below. Then we should be good to merge! 😄